Controllers/CouriersController.go

```diff
@@ -0,0 +1,29 @@
+package Controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Baraulia/COURIER_SERVICE/Models"
+	"github.com/Baraulia/COURIER_SERVICE/db"
+	"net/http"
+)
+
+var Couriers []db.SmallInfo
+
+func GreetingPage(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Hello! This is a courier delivery page")
+}
+
+func GetCouriers(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	Couriers = Models.GetCouriers(Couriers)
+	json.NewEncoder(w).Encode(Couriers)
+}
+
+func GetOneCourier(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	var Courier db.SmallInfo
```

Comment:
This is not good parsing query parameters directly in db objects. In group project please try to do your best to keep layers independent from each other. Even if it requires a bit of overhead of copying data from one structures to another.

Reply:
Oh, well, sorry, I slightly misunderstood the usage of this object. Then I have a different question. Why do you need to pass empty courier as function parameter? why it can't just be `Courier := Models.GetOneCourier(id)`

Reply:
@TimofeiShorokhov please fix all other places like this also, e.g. `GetCouriers`, `GetOneCourierFromDB`, etc